src/components/Profile.jsx

```diff
@@ -1,16 +1,50 @@
-import React from "react";
-import Sidebar from "./Sidebar";
-import ProfilePageTop from "./profilePageTop/ProfilePageTop";
-import styled from "styled-components";
+import React, {useContext, useState, useEffect} from 'react';
+import Sidebar from './Sidebar';
+import ProfilePageTop from './profilePageTop/ProfilePageTop';
+import styled from 'styled-components';
+import {AuthContext} from '../contexts/AuthContext';
+import {LocationContext} from '../contexts/LocationContext';
+import firebase from '../plugins/firebase';
+import ShoutComponent from './ShoutComponent';
 
 const Profile = () => {
+  const {auth} = useContext(AuthContext);
+  const {getLocation} = useContext(LocationContext);
+  const [shoutData, setShoutData] = useState([]);
+
+  getLocation('/profile');
+
+  useEffect(() => {
+    let shouts = [];
+    firebase
+      .firestore()
+      .collection('shouts')
+      .where('userId', '==', auth.uid)
+      .orderBy('createdAt', 'desc')
+      .get()
+      .then(querySnapshot => {
+        querySnapshot.forEach(doc => {
+          shouts.push({
+            data: doc.data(),
+            id: doc.id,
+          });
+        });
+        setShoutData(shouts);
+        console.log('user shouts: ', shouts);
+      })
+      .catch(err => console.log(err));
+  }, [auth.uid]);
```

Comment:
I am just curious to know why you put `auth.uid` 😄

Reply:
haha, my mistake...I changed it to shoutData on my latest branch